api/src/main/java/com/codeforcommunity/dto/site/GetSiteResponse.java

```diff
@@ -13,6 +13,7 @@ public class GetSiteResponse {
   private final String address;
   private final Integer neighborhoodId;
   private final List<SiteEntry> entries;
+  private final java.sql.Date plantingDate;
```

Comment:
:wrench: Similar to the comment above, can we move `plantingDate` to `SiteEntry.java` (src/main/java/com/codeforcommunity/dto/site/SiteEntry.java) so that a site can have multiple planting dates?